packages/dd-trace/src/config.js

```diff
@@ -521,6 +521,18 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       true
     )
 
+    const DD_EXPERIMENTAL_API_SECURITY_ENABLED = coalesce(
+      options.appsec?.apiSecurity?.enabled,
+      process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED && isTrue(process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED),
+      false
+    )
+
+    const DD_API_SECURITY_REQUEST_SAMPLE_RATE = coalesce(
+      options.appsec?.apiSecurity?.requestSampling,
+      parseFloat(process.env.DD_API_SECURITY_REQUEST_SAMPLE_RATE),
+      0.1
+    )
```

Comment:
not an amazingly consistent env var naming :/

Reply:
are you suggesting to change the config property to something like:
`appsec.apiSecurity.requestSampleRate`
?

Reply:
no sorry I meant the env vars, one is experimental but the other one is not ? a bit weird imo

Reply:
RFC stuff